__tests__/index.test.js

```diff
@@ -194,3 +194,51 @@ describe('GET /api/articles',()=>{
 });
 
 
+describe('GET /apri/articles (topic query',()=>{
+    test('return a 200 code when a request is made to the api',()=>{
```

Comment:
We technically don't need this test block as we have an .expect(200) in the below test too rendering this one a little obsolete.